src/utils/checkItem.js

```diff
@@ -0,0 +1,3 @@
+export const checkItem = (array, id) => {
+  return !!array?.some((item) => item._id === id)
```

Comment:
Did not get the use of `!!` here

Reply:
Oh yeah I can remove !! as I am using array.some